tests/xml/sirtfi_v2/bad_sirtfi_no_email_in_contact.yaml

```diff
@@ -1,7 +1,12 @@
 expected:
-- status: error
-  component_id: check_sirtfi
-  message: SIRTFI requires a REFEDS security contact with an EmailAddress
-- status: error
-  component_id: check_sirtfi2
-  message: Sirtfi version 2 requires a REFEDS security contact to have an EmailAddress
+  - status: error
+    component_id: check_sirtfi
+    message: SIRTFI requires a REFEDS security contact with an EmailAddress
+  - status: error
+    component_id: check_sirtfi2
+    message: Sirtfi version 2 requires a REFEDS security contact to have an EmailAddress
+# Skip this test for now 
+override:
+  - endpoint: [v09, v09x, v010, v010x]
+    skip: true

```

Comment:
This seems over-complicated. The following would suffice:

```yaml
# Skip this test for now
skip: true
```

The `skip` doesn't need to be in an override, in other words.

Reply:
Ah! I did not know I could just use a skip property like that, but you can. That said, I quite liked using it as an override because it preserved the original expected error messages in the file. This way, when we need to include these tests, we can simply remove the override.

Reply:
What I meant was that you can have both `expected` and `skip` at the top level. The `skip` is looked at first and then the `expected` is ignored. So you retain the expected error messages without using `override` (and without having to enumerate all the endpoints).

This is definitely not critical, but it's a change you can make if you like.

Reply:
'Ah, I see. That is indeed cleaner. I will do that and then merge it in.'. This is now done.